neat-form-core/src/test/java/com/nerdstone/neatformcore/robolectric/builders/DateTimePickerViewBuilderTest.kt

```diff
@@ -97,6 +98,20 @@ class DateTimePickerViewBuilderTest : BaseJsonViewBuilderTest() {
         Assert.assertTrue(dateTimePickerViewBuilder.textInputEditText.text.toString() == "01/01/2019")
     }
 
+    @Test
+    fun `Should return date from passed pattern`() {
+        val today = Calendar.getInstance()
+        today[Calendar.HOUR_OF_DAY] = 12
+        today[Calendar.MINUTE] = 0
+        today[Calendar.SECOND] = 0
+        today[Calendar.MILLISECOND] = 0
+
+        Assert.assertEquals(today.timeInMillis,dateTimePickerViewBuilder.getDate("today"))
+
+        today.add(Calendar.YEAR,1)
+        Assert.assertEquals(today.timeInMillis,dateTimePickerViewBuilder.getDate("today+1y"))
+    }
+
```

Comment:
You may need to add more test cases for months & days to cover the decreased coverage.

Reply:
Check the affected class in the comment  made by coveralls